colorfield/validators.py

```diff
@@ -17,3 +17,50 @@
     _("Enter a valid hexa color, eg. #00000000"),
     "invalid",
 )
+
+COLOR_RGB_RE = re.compile(
+    # prefix and opening parenthesis
+    r"^rgb\("
+    # first number
```

Comment:
Considering that we keep comments here, they could be more descriptive like `# first number: red channel` and so on...